custom_components/sat/coordinator.py

```
@@ -90,18 +107,51 @@ def device_type(self) -> str:
+        """Return the current status of the device."""
+        if self.boiler_temperature is None:
+            return DeviceStatus.INITIALIZING
 
-    @property
-    def manufacturer(self) -> Manufacturer | None:
-        if self.member_id is None:
-            return None
+        if self.hot_water_active:
+            return DeviceStatus.HOT_WATER
+
+        if self.setpoint is None or self.setpoint <= MINIMUM_SETPOINT:
```

Ruff (0.8.2)

`MINIMUM_SETPOINT` may be undefined, or defined from star imports

(F405)